### PR TITLE
Fix  #109322 vogue.co.kr 

### LINKED ISSUE #109322: vogue.co.kr
### Issue URL (Social Widget)
 [https://www.vogue.co.kr/2022/01/29/%EC%9A%94%EC%A6%98-%EB%96%A0%EC%98%A4%EB%A5%B4%EB%8A%94-%EB%8B%88%ED%8A%B8%EC%9B%A8%EC%96%B4-%EB%B8%8C%EB%9E%9C%EB%93%9C-7/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.vogue.co.kr%2F2022%2F01%2F29%2F%25EC%259A%2594%25EC%25A6%2598-%25EB%2596%25A0%25EC%2598%25A4%25EB%25A5%25B4%25EB%258A%2594-%25EB%258B%2588%25ED%258A%25B8%25EC%259B%25A8%25EC%2596%25B4-%25EB%25B8%258C%25EB%259E%259C%25EB%2593%259C-7%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.6.17
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental

### PR DESCRIPTION
**Pull request by piquark6046**
#109322

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com|bolgehastanesi.com
 !
+||common.doosanmagazine.com/reciveLogs.php
```
Comment (Sergey-Lyapin) [inline SpywareFilter/sections/specific.txt:50]: What about:
`||vogue.co.kr/wp-content/themes/Avada-Child-Theme/js/GA.min.js`
instead

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt:50]: It looks not bad.
However, the rule can be useful on other website (e.g. https://www.allurekorea.com/ ).

Comment (Sergey-Lyapin) [reply SpywareFilter/sections/specific.txt:50]: @piquark6046 
with this rule `/Avada-Child-Theme/js/GA.*js$domain=allurekorea.com|vogue.co.kr`
_request common.doosanmagazine.com/reciveLogs.php_ doesn't exist so let's replace `||common.doosanmagazine.com/reciveLogs.php` with `/Avada-Child-Theme/js/GA.*js$domain=allurekorea.com|vogue.co.kr`.

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt:56]: Pushed.

**Pull request merged**